src/components/Modals/GettingStartedModal.tsx

```diff
@@ -44,7 +44,7 @@ type Level = {
   footerButton: { title: string; cta: any };
 };
 
-export const LOCAL_STORAGE_KEY = 'hacbs/getting-started-modal';
+export const LOCAL_STORAGE_KEY = 'getting-started-modal';
```

Comment:
The app studio modal is also using the same local storage key.

Reply:
@rottencandy There is a ticket and a [PR](https://github.com/openshift/hac-dev/pull/285) to remove GettingStartedModals completely. So i think we should fine with this now

Reply:
Okay sounds good

Reply:
This might be the reason why e2e tests are getting stuck on getting started modal.

Reply:
I can put it back for now and let it be removed if/when they are completely removed.

Reply:
> This might be the reason why e2e tests are getting stuck on getting started modal.

No, when i checked locally cypress is able to click on the Create Application button behind the modal and proceeded with the flow. I think the issue is, Applications is not loading in the ephemeral env, you can see this issue in Abhi's PR where he removed this modal completely but still e2e is failing with that error

![image](https://user-images.githubusercontent.com/9964343/208122066-2b3c2394-9ca0-4d89-be26-557d5da5a718.png)